snap/snapcraft.yaml

```diff
@@ -120,9 +120,17 @@ parts:
       cp -v completion.bash ../install
       # python3 fixup symlink (snapcraft bug)
       ln -sf ../usr/bin/python3.8 $SNAPCRAFT_PART_INSTALL/bin/python3
+      mkdir -p $SNAPCRAFT_PART_INSTALL/meta/hooks
+      mv $SNAPCRAFT_PART_INSTALL/bin/charmcraft-*-hook $SNAPCRAFT_PART_INSTALL/meta/hooks
     organize:
       # move things around so they're tidier
       usr/lib/git-core: git/git-core
       usr/share/git-core/templates: git/templates
       usr/bin/git: bin/git
       lib/python3.8/site-packages: lib/
+
+hooks:
+  configure:
+    passthrough:
+      environment:
+        PATH: "$SNAP/bin:$PATH"
```

Comment:
given the opportunity (this is it), I would restrict the path to just the things we need as this is a classic confined snap

Reply:
Good point, will fix!

Reply:
Done!